### ISSUE TITLE
Add NSIS build to appveyor

### ISSUE THREAD

**Issue by phlptp** [GMLC-TDC/HELICS-src#221]
I noticed Appveyor had NSIS installed.  Is there such a thing as a daily build on Appveyor that could run cpack on the develop branch and have the resulting installer available as an artifact?

**Comment by nightlark** [comment]
If the build succeeds but the tests fail, do we still want the installer created?

**Comment by nightlark** [comment]
It could be uploaded to GitHub Releases or BinTray.

**Comment by phlptp** [comment]
what is binTray?

**Comment by nightlark** [comment]
It is for distribution of artifacts, similar to GitHub Releases. My impression of the difference is that it seems to have a bigger focus on handling multiple releases for different platforms/package formats, and easier to see past versions that have been released.

The same company behind Bintray also makes Artifactory, which seems more oriented towards development artifacts coming out of Travis/Appveyor, but it seems like the bar for OSS projects getting access to Artifactory is higher.

**Comment by phlptp** [comment]
@nightlark @kdheepak   can you and anyone else with an interest in this dig a little more and make a recommendation on the hosting or other processes regarding nightly build artifacts and how to use the artifacts in appveyor.  I assume we will attach some installer files to the release in github, but I would actually want those to include the matlab binaries so will probably need to be generated separately.

**Comment by nightlark** [comment]
#240 adds an NSIS installer as an artifact to Appveyor builds. Appveyor seems more limited than Travis in terms of having separate configurations for cron/daily builds, but a check could be added to see if it is a scheduled/cron-triggered build if we want to limit the creation of an installer to a daily Appveyor build.

**Comment by phlptp** [comment]
Do we have a daily build on appveyor?  if so only building the cpack stuff on the daily build for develop would be sufficient for what we need it for.  Otherwise I only really care about it on the develop branch

**Comment by nightlark** [comment]
No, we don't have a daily build on appveyor. I emailed support to see if they can enable scheduled builds, and asked about getting 2 concurrent builds, which I think they'll decline -- but maybe we will get lucky.
